src/app/core/data/metadata-value-data.service.spec.ts

```diff
@@ -0,0 +1,109 @@
+import {Observable, of as observableOf} from 'rxjs';
```

Comment:
what is this test doing?
ADD this description into every test

Reply:
doc added